src/state/session/index.tsx

```diff
@@ -209,6 +207,7 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
   const clearCurrentAccount = React.useCallback(() => {
     logger.warn(`session: clear current account`)
     __globalAgent = PUBLIC_BSKY_AGENT
+    BskyAgent.configure({appLabelers: [BSKY_LABELER_DID]})
```

Comment:
This slipped in with modservices. I think this should be reset when the account is logged out.